### PR TITLE
Release 34.0.0

### PR DESCRIPTION
**Pull request by mcmire**
This release includes initial releases for all new packages.

---

The release notes will end up being something like:

> # 34.0.0
> 
> ## @metamask/announcement-controller
> 
> ### Added
> - Initial release
>   - As a result of converting our shared controllers repo into a monorepo ([#831](https://github.com/MetaMask/controllers/pull/831)), we've created this package from select parts of [`@metamask/controllers` v33.0.0](https://github.com/MetaMask/controllers/tree/v33.0.0), namely:
>     - `src/user/AddressBookController.ts`
>     - `src/user/AddressBookController.test.ts`
>
>     All changes listed after this point were applied to this package following the monorepo conversion.
> 
> ## @metamask/address-book-controller
> 
> ### Added
> - Initial release
>   - As a result of converting our shared controllers repo into a monorepo ([#831](https://github.com/MetaMask/controllers/pull/831)), we've created this package from select parts of [`@metamask/controllers` v33.0.0](https://github.com/MetaMask/controllers/tree/v33.0.0), namely:
>     - Everything in `src/announcement`
>
>     All changes listed after this point were applied to this package following the monorepo conversion.
>
> ...

### CODE REVIEW

**Comment by mcmire** [comment]
Currently blocked on https://github.com/MetaMask/auto-changelog/pull/130.

**Comment by mcmire** [comment]
Blocked on https://github.com/MetaMask/auto-changelog/pull/131.

File: package.json
Comment (Gudahtt) [inline package.json:3]: Should this be `-rc.1` as well?

Comment (mcmire) [reply package.json]: Hmm... Yes, I think that makes sense given this is a special release. Updated in [0e5cd32](https://github.com/MetaMask/controllers/pull/960/commits/0e5cd329f2ff7c07a7d0c3cdbc7549f3a56a3153).

**Comment by mcmire** [comment]
Putting this back in draft until we are truly ready to merge this.

**Comment by mcmire** [comment]
@metamaskbot publish-preview

**Comment by mcmire** [comment]
Testing new packages out on extension: https://github.com/MetaMask/metamask-extension/pull/16547

**Comment by mcmire** [comment]
@metamaskbot publish-preview

File: packages/assets-controllers/src/index.ts
Comment (mcmire) [inline packages/assets-controllers/src/index.ts:11]: This has been extracted to https://github.com/MetaMask/controllers/pull/973 and will remain here until that's merged.

**Comment by Gudahtt** [comment]
@rickycodes pointed out to me today that this line will probably fail when this is merged, because it's no longer true that release commits are made by GitHub actions: https://github.com/MetaMask/controllers/blob/main/.github/workflows/main.yml#L30

**Comment by mcmire** [comment]
@metamaskbot publish-preview

**Review by Gudahtt — APPROVED** [review]
LGTM!

All of the changelogs/versions look good, and everything seems to work on the extension

**Pull request merged**
